databricks/koalas/tests/test_series.py

```diff
@@ -1775,6 +1775,17 @@ def test_mod(self):
         kdf = ks.from_pandas(pdf)
         self.assert_eq(kdf.a.mod(kdf.b), pdf.a.mod(pdf.b))
 
+    def test_mode(self):
+        pser = pd.Series([0, 0, 1, 1, 1, np.nan, np.nan, np.nan])
+        kser = ks.from_pandas(pser)
+        self.assert_eq(kser.mode(), pser.mode())
+        self.assert_eq(kser.mode(False).sort_values().values, pser.mode(False).sort_values().values)
```

Comment:
Seems like pandas < 0.24 doesn't support any parameter for `Series.mode()`

Maybe we can separate the test.

Reply:
Thank you! Modified.